examples/debouncing/src/main/scala/example/Main.scala

```diff
@@ -0,0 +1,67 @@
+package example
+
+import cats.effect.IO
+import tyrian.Html.*
+import tyrian.*
+
+import scala.concurrent.duration.*
+import scala.scalajs.js.annotation.*
+
+@JSExportTopLevel("TyrianApp")
+object Main extends TyrianIOApp[Msg, Model]:
+
+  val DEBOUNCING_MILLIS: Int = 500
+  val TICK_INTERVAL: Int = 100
+
+  def router: Location => Msg = Routing.none(Msg.NoOp)
+
+  def init(flags: Map[String, String]): (Model, Cmd[IO, Msg]) =
+    (Model("", None), Cmd.None)
+
+  def update(model: Model): Msg => (Model, Cmd[IO, Msg]) =
+    case Msg.UpdateValue(v) =>
+      (model.copy(debouncer = Some(v, DEBOUNCING_MILLIS)), Cmd.None)
+    case Msg.TimePassed =>
+      model.debouncer match
+        case Some((v, time)) =>
+          if (time < 0)
+            (model.copy(value = v), Cmd.None)
```

Comment:
I believe this should also reset the debouncer:
`model.copy(value = v, debouncer = None)`

Reply:
True, I've forgotten about it. Fixed.